tests/example-premature-termination/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+full_path=$(readlink -e "${BASH_SOURCE[0]}" )
+script_dir=${full_path%/*}
+slug=${script_dir##*/}
+
+if cobolcheck_type=$(type cobolcheck); then
+    echo "Found cobolcheck, ${cobolcheck_type}"
+    COBOLCHECK=cobolcheck
+elif [[ ! -x $SCRIPT_DIR/bin/cobolcheck ]]; then
+    echo "cobolcheck not found, try to fetch it."
+    ./bin/fetch-cobolcheck
+fi
+
+cd $script_dir
+"${COBOLCHECK}" -p "${slug}"
```

Comment:
In the `else` case, `COBOLCHECK` doesn't get set. Should it use `$SCRIPT_DIR/bin/cobolcheck`?

```suggestion
if cobolcheck_type=$(type cobolcheck); then
    echo "Found cobolcheck, ${cobolcheck_type}"
    COBOLCHECK=cobolcheck
else
    COBOLCHECK=$SCRIPT_DIR/bin/cobolcheck
fi
if [[ ! -x $COBOLCHECK ]]; then
    echo "cobolcheck not found, try to fetch it."
    ./bin/fetch-cobolcheck
fi

cd "${script_dir}"
"${COBOLCHECK}" -p "${slug}"
```

Reply:
Your suggested change for fixing undefined COBOLCHECK has been incorporated.